### ISSUE TITLE
Reduced font size on abbr?

### ISSUE THREAD

**Issue by pdeffendol** [zurb/foundation#6658]
Is there a justification for reducing the font-size on `<abbr>` elements?  Maybe this should be a configurable setting (for SASS)?

```
    /* Abbreviations */
    abbr,
    acronym {
      text-transform: uppercase;
      font-size: 90%;
      color: $body-font-color;
      cursor: $cursor-help-value;
    }
    abbr {
      text-transform: none;
      &[title] {
        border-bottom: $acronym-underline;
      }
    }
```

**Comment by rafibomb** [comment]
I don't know what the reason is or when I'd want smaller. We'll accept a PR to delete that property. Thanks!

**Comment by pdeffendol** [comment]
Merged in #6692
